Changelog.md

```diff
@@ -1,14 +1,12 @@
-0.10.10.1 <ietf-dane@dukhovni.org> May 2020
+0.10.10.1 <ietf-dane@dukhovni.org> June 2020
```

Comment:
I was unsure what to do about the email address, since the release now seems like such a collaborative effort. How about simply removing it?

```suggestion
0.10.10.1 – June 2020
```

Reply:
Works for me.